### ISSUE TITLE
Suggestion: Add option to limit how often the map renders

### ISSUE THREAD

**Issue by OakLoaf** [granny/Pl3xMap-test#16]
It would be nice to be able to set how often the map renders. For example being able to set it to full render once every 30 minutes

**Comment by jedrek0429** [comment]
I don't think there's any reason to rerender the map periodically. You can create your own automation scripts that run the command in your server console if you want.

**Comment by OakLoaf** [comment]
> I don't think there's any reason to rerender the map periodically. You can create your own automation scripts that run the command in your server console if you want.

I more so want to it as a limit not as a schedule. Currently there is no way to stop the map from rerendering frequently

**Comment by jedrek0429** [comment]
Yeah, okay, that makes sense. I didn’t realise you wanted to _stop_ the rendering.
